hack/jenkins/gotest-dockerized.sh

```diff
@@ -36,6 +36,9 @@ export KUBE_JUNIT_REPORT_DIR=${WORKSPACE}/_artifacts
 # From _inside_ a container that has the host's docker mapped in, the $PATH
 # provided must be resolvable on the *HOST*, not the container.
 
+# TODO: either push docker people to fix --rm=true option or clean up images after tests
```

Comment:
I would argue against putting that into the test script because I don't know how well docker will take to it being executed in parallel across multiple runs.